plugins/css-prefers-color-scheme/test/_browser.mjs

```diff
@@ -0,0 +1,158 @@
+import puppeteer from 'puppeteer';
```

Comment:
Shouldn't this be listed as `devDependencies` of this package?

Reply:
yup!